homeassistant/components/mystrom/strings.json

```
@@ -17,5 +17,12 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "avg_consumption": {
+        "name": "Average Consumption"
```

I think the second word should be lower-case to follow [HA style](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/).
```suggestion
        "name": "Average consumption"
```

"Entity names all start with a capital letter, the rest of the words are lower case"